docs/ref/release_notes.rst

```diff
@@ -1,6 +1,42 @@
 Release Notes
 =============
 
+
+What's new in iris-grib v0.15
+-----------------------------
+
+:Release: 0.15.0
+:Date: 4 Dec 2019
+
+Features
+^^^^^^^^
+
+* Updated translations between GRIB parameter code and CF standard_name or
+  long_name
```

Comment:
This isn't of much value to anyone as it stands, as it doesn't provide any way to see what actually changed.
```suggestion
  long_name :
      * additional WAFC codes, both to and from CF
      * 'mass_fraction_of_cloud_liquid_water_in_air' and 'mass_fraction_of_cloud_ice_in_air', both to and from CF
      * 'surface_downwelling_longwave_flux_in_air', now translates to GRIBcode(2, 0, 5, 3)  (but not the reverse).
      * for full details, see : https://github.com/Scitools/iris-grib/compare/c4243ae..5c314e3#diff-cf46b46880cae59e82a91c7ab6bb81ba
```